applications/rag/example_notebooks/rag-data-ingest-with-kubernetes-docs.ipynb

```diff
@@ -0,0 +1,286 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# RAG-on-GKE Application\n",
+        "\n",
+        "This is a Python notebook for generating the vector embeddings based on [Kubernetes docs](https://github.com/dohsimpson/kubernetes-doc-pdf/) used by the RAG on GKE application.   \n",
+        "For full information, please checkout the GitHub documentation [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/rag/README.md).\n",
+        "\n",
+        "\n",
+        "\n",
+        "- Clone the kubernetes docs repo\n",
```

Comment:
Should this be in a separate cell? Besides, can you change the format of these sub titles to headings? i.e. `## Clone the kubernetes docs repo\n`. Same for all the below sub titles